packages/flutter_tools/test/src/common.dart

```diff
@@ -145,6 +145,15 @@ Future<void> expectToolExitLater(Future<dynamic> future, Matcher messageMatcher)
   }
 }
 
+Matcher containsIgnoringWhitespace(String toSearch){
```

Comment:
The indentation here doesn't match our [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting) (also typo in `toSearcCollapsedWhitespaces`. It should be something like (please double-check, coding in github is hard):

```dart
Matcher containsIgnoringWhitespace(String toSearch) {
  return predicate(
    (String source)  {
      final String sourceCollapsedWhitespaces = collapseWhitespace(source);
      final String toSearchCollapsedWhitespaces = collapseWhitespace(toSearch);
      return sourceCollapsedWhitespaces.contains(toSearchCollapsedWhitespaces);
    },
    'contains "$toSearch" ignoring whitespaces.',
  );
}
```

Reply:
I simplified the code and applied the styling according to the guide.